tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "declaration": true,
     "declarationDir": "dist/types",
     "forceConsistentCasingInFileNames": true,
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "downlevelIteration": true
```

Comment:
This seems unnecessary

Reply:
TS complaints because the target is ES5. Makes sense. If we abandon support for IE, we can target es6 and remove this.